src/components/row/row.stories.mdx

```diff
@@ -10,7 +10,7 @@ import LinkTo from "@storybook/addon-links/react";
 <Meta title="Row" parameters={{ info: { disable: true } }} />
 
 <DeprecationWarning>
-  Row and Column are due to be depricated. Please use the{" "}
+  Row and Column are both now deprecated. Please use the{" "}
```

Comment:
I probably would leave this as is because Kuba is working on the deprecation of Row and Column.